aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -414,6 +414,14 @@ Tensor NestedTensor_to_padded_tensor_generic(
   return ret_val;
 }
 
+// alias for to_padded_tensor in nested namespace
```

Comment:
Going to go and refactor the nestedtensor math file and split to smaller more logical cpp files. Hence the comment above

Reply:
Added NestedTensorAliases.cpp, will import and include in the relevant bzl files